index.html

```diff
@@ -848,12 +848,14 @@
   <dd>
     Because the [=user agent=] is a [=trustworthy agent=], it is held to be loyal to the
     [=person=] using it in all situations, including in preference to the [=user agent=]'s implementer.
```

Comment:
We have an interesting case study for this in the holdback proposal for https://github.com/RupertBenWiser/Web-Environment-Integrity/blob/main/explainer.md. Let's chat about that in today's meeting.